### DISCUSSION TITLE
What city are The 🪲🪲🪲🪲 from?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#945] [Galaxy Trivia] [answered]
**Bonus:** Sort original members by DOB (desc).

**Comment by tomerf** [comment, marked as answer]
The Beatles are from Liverpool, England.
The members include:
1. George Harrison - Born on February 25, 1943
1. Paul McCartney - Born on June 18, 1942
1. John Lennon - Born on October 9, 1940
1. Ringo Starr - Born on July 7, 1940

**Comment by peterbaccaro** [comment]
**The Beatles are from Liverpool, England**. Liverpool is a city located in the northwest part of England and is famous for being the birthplace of the Beatles. The band members, John Lennon, Paul McCartney, George Harrison, and Ringo Starr, all grew up in Liverpool and began their musical journey there. Liverpool has since become synonymous with the Beatles and attracts many fans and tourists from around the world.

Here are the original members of The Beatles sorted by their dates of birth (desc):

- **George Harrison** - Born on February 25, 1943
- **Paul McCartney** - Born on June 18, 1942
- **John Lennon** - Born on October 9, 1940
- **Ringo Starr** (real name: Richard Starkey) - Born on July 7, 1940
